src/components/UserManagement/UserTableData.jsx

```diff
@@ -99,25 +107,31 @@ const UserTableData = React.memo(props => {
       <td>{props.user.endDate ? formatDate(props.user.endDate) : 'N/A'}</td>
       {checkPermissionsOnOwner() ? null : (
         <td>
+        {
+          props.auth.user.userid === props.user._id || props.auth.user.role === "Owner" && 
+           props.user.role === "Administrator"  ? '': 
```

Comment:
```suggestion
          props.auth.user.userid === props.user._id ? '': 
```
Is this PR specifically to prevent admin and owners from deleting themselves and other roles with permission to delete accounts should be able to delete themselves? Or should everyone (with permission to delete accounts) not be able to delete themselves?

Reply:
This also prevents ANY admin from being deleted which is definitely not the desired behavior